sql-statements/sql-statement-show-status.md

```diff
@@ -5,56 +5,66 @@ summary: An overview of the usage of SHOW [GLOBAL|SESSION] STATUS for the TiDB d
+A full description of the variables can be found here: [status variables](/status-variables.md)
 
-**ShowTargetFilterable:**
+## Synopsis
 
-![ShowTargetFilterable](/media/sqlgram/ShowTargetFilterable.png)
+```ebnf+diagram
+ShowStatusStmt ::=
+    'SHOW' Scope? 'STATUS' ShowLikeOrWhere?
 
-**GlobalScope:**
+Scope ::=
+    ( 'GLOBAL' | 'SESSION' )
 
-![GlobalScope](/media/sqlgram/GlobalScope.png)
+ShowLikeOrWhere ::=
+    "LIKE" SimpleExpr
+|   "WHERE" Expression
+```
 
 ## Examples
 
 ```sql
-mysql> show status;
-+--------------------+--------------------------------------+
-| Variable_name      | Value                                |
-+--------------------+--------------------------------------+
-| Ssl_cipher_list    |                                      |
-| server_id          | 93e2e07d-6bb4-4a1b-90b7-e035fae154fe |
-| ddl_schema_version | 141                                  |
-| Ssl_verify_mode    | 0                                    |
-| Ssl_version        |                                      |
-| Ssl_cipher         |                                      |
-+--------------------+--------------------------------------+
-6 rows in set (0.01 sec)
-
-mysql> show global status;
-+--------------------+--------------------------------------+
-| Variable_name      | Value                                |
-+--------------------+--------------------------------------+
-| Ssl_cipher         |                                      |
-| Ssl_cipher_list    |                                      |
-| Ssl_verify_mode    | 0                                    |
-| Ssl_version        |                                      |
-| server_id          | 93e2e07d-6bb4-4a1b-90b7-e035fae154fe |
-| ddl_schema_version | 141                                  |
-+--------------------+--------------------------------------+
-6 rows in set (0.00 sec)
+mysql> SHOW SESSION STATUS;
++-------------------------------+--------------------------------------+
+| Variable_name                 | Value                                |
++-------------------------------+--------------------------------------+
+| Ssl_cipher                    |                                      |
+| Ssl_cipher_list               |                                      |
+| Ssl_server_not_after          |                                      |
+| Ssl_server_not_before         |                                      |
+| Ssl_verify_mode               | 0                                    |
+| Ssl_version                   |                                      |
+| Uptime                        | 1409                                 |
+| ddl_schema_version            | 116                                  |
+| last_plan_binding_update_time | 0000-00-00 00:00:00                  |
+| server_id                     | 61160e73-ab80-40ff-8f33-27d55d475fd1 |
++-------------------------------+--------------------------------------+
+10 rows in set (0.00 sec)
+
+sql> SHOW GLOBAL STATUS;
```

Comment:
```suggestion
mysql> SHOW GLOBAL STATUS;
```

Reply:
I have overlooked this when I reviewed https://github.com/pingcap/docs/pull/15363 .